src/components/side-panel/panel-header.js

```diff
@@ -59,6 +59,7 @@ const StyledPanelAction = styled.div.attrs({
   height: 26px;
   justify-content: space-between;
   margin-left: 4px;
+  ${props => props.hasLabel ? 'width: 70px' : ''}
```

Comment:
I recall doing a very similar change on my backend storage branch, and we might not need this if they changed their design :)

Reply:
Yes, this change appears because of a conflict, and you're right, it is not needed anymore, so I'll remove it.